src/app/pages/datasets/components/dataset-form/dataset-form.component.ts

```diff
@@ -130,7 +131,7 @@ export class DatasetFormComponent implements FormConfiguration {
       label: true,
       config: [
         {
-          type: 'input',
+          type: 'textarea',
```

Comment:
<img width="804" alt="image" src="https://user-images.githubusercontent.com/351613/195798919-92488262-62b1-4fa9-b81d-b2108162bd0b.png">

<img width="806" alt="image" src="https://user-images.githubusercontent.com/351613/195799154-9f433cda-05bf-40fb-98a8-3691b10aef4a.png">

`Add Dataset` and `Edit Dataset` forms look inconsistently

Reply:
<img width="803" alt="image" src="https://user-images.githubusercontent.com/351613/196138144-a6ef4704-9bc9-49d1-8114-48899ca146a9.png">

This example is when the dataset name reaches the limit. Perhaps three lines in the textarea would be enough in most cases and help to save some vertical space, especially in the `Add dataset` form with two textarea fields. Does it make sense?